src/js/components/Box/stories/RTL.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import { Grommet, Box } from 'grommet';
+import { grommet } from '../../../themes';
+
+const RTLBox = () => (
+  <Grommet theme={grommet} dir="rtl">
```

Comment:
On `Grommet.js` the `dir` prop is referred to as `direction`, I'm not sure which one is true since this prop isn't currently documented. 
I think we should stick with `direction` rather than `dir` (`direction` sounds more grommet), and add it to Grommet/doc.js as a new documented prop.

Reply:
Hmm... Let's discuss. `dir="rtl"` is the HTML standard way to describe it. I could see going either direction though. :)

Reply:
I think we can continue with `dir="rtl"` as discussed.